src/clients/aggregator_client/api_types.rs

```diff
@@ -188,12 +189,20 @@ pub struct TaskIds {
     pub pagination_token: Option<String>,
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone, Copy)]
+#[derive(Deserialize, Serialize, Debug, Clone, Copy, PartialEq, Eq)]
 pub struct TaskMetrics {
     pub reports: u64,
     pub report_aggregations: u64,
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct AggregatorApiConfig {
+    pub dap_url: Url,
+    pub role: AggregatorRole,
+    pub vdafs: Vec<u32>,
+    pub query_types: Vec<u8>,
```

Comment:
I'd love to see enums identifying VDAFs and query types instead of `u32` and `u8`, though I'm not sure off the top of my head where the best place to define those is.

Reply:
I'll add that to this PR shortly

Reply:
addressed in bdd0e1a